### PR TITLE
Added bazel target for aes_interrupt_encryption

### PR DESCRIPTION
**Pull request by rprakas-gsc**
Updating the aes_testplan.json file with bazel targets for Aes_interrupt_encryption_Test

Adding @jwnrt as reviewer

### CODE REVIEW

File: hw/top_earlgrey/data/ip/chip_aes_testplan.hjson [hunk 1]
```diff
@@ -111,7 +111,8 @@
       stage: V2
       si_stage: SV3
       lc_states: ["PROD"]
-      tests: []
+      tests: ["chip_sw_aes_interrupt_encryption"]
```
Comment (engdoreis) [inline hw/top_earlgrey/data/ip/chip_aes_testplan.hjson:114]: This field is used for to link dv tests listed in the file `hw/top_earlgrey/dv/chip_sim_cfg.hjson`.
So in this case does not apply, can you please drop this change?

**Review by engdoreis — CHANGES_REQUESTED** [review]

File: hw/top_earlgrey/data/ip/chip_aes_testplan.hjson (hunk 1 above)
Comment (rprakas-gsc) [reply hw/top_earlgrey/data/ip/chip_aes_testplan.hjson]: Sure.  Just updated the file

File: hw/top_earlgrey/data/ip/chip_aes_testplan.hjson [hunk 2]
```diff
@@ -112,6 +112,7 @@
       si_stage: SV3
       lc_states: ["PROD"]
       tests: []
+      bazel: ["//sw/device/tests/aes_interrupt_encryption_test"]
```
Comment (jwnrt) [inline hw/top_earlgrey/data/ip/chip_aes_testplan.hjson:115]:
```suggestion
      bazel: ["//sw/device/tests:aes_interrupt_encryption_test"]
```

**Comment by rprakas-gsc** [comment]
@engdoreis - I have resolved the conflict, and force pushed the changes

**Comment by engdoreis** [comment]
@rprakas-gsc the PR filed the linting because of the commit message. You have to squash both commits into one and make sure that the massage follows the [style gude](https://opentitan.org/book/doc/contributing/detailed_contribution_guide/index.html?highlight=commit%20message#what-should-commit-messages-look-like).

**Pull request merged**